qstnrs/README.rst

```diff
@@ -0,0 +1,23 @@
+Questionnaire
+=============
+
+A Django application where authenticated users can create questionnaires that can be filled in by
+unauthenticated users.
+
+Quick start
+-----------
+1. Install the Questionnaire egg file by running:
+    easy_install Questionnaire.egg
```

Comment:
I do not prefer one over the other. 
I found out about easy_install when I was reading about creating and installing egg files (http://peak.telecommunity.com/DevCenter/PythonEggs).
I'll change it to use pip.